tests/installation/bootloader_hyperv.pm

```diff
@@ -15,15 +15,7 @@ use strict;
 use warnings;
 use File::Basename;
 use data_integrity_utils 'verify_checksum';
-
-sub hyperv_cmd {
-    my ($cmd, $args) = @_;
-    $args->{ignore_return_code} ||= 0;
-    my $ret = console('svirt')->run_cmd($cmd);
-    diag "Command on Hyper-V returned: $ret";
-    die 'Command on Hyper-V failed' unless ($args->{ignore_return_code} || !$ret);
-    return $ret;
-}
+use virt_autotest::hyperv_utils 'hyperv_cmd';
 
 sub hyperv_cmd_with_retry {
```

Comment:
hyperv_cmd_with_retry belongs to hyperv_utils.pm ???

Reply:
I don't use it this time, if next time use it in other place, will add it to hyperv_utils.pm.

Reply:
Why next time ??? It looks to me it should be moved to hyperv_utils.pm obviously.

Reply:
all right, I will add it to utils.

Reply:
Done